external-crates/move/crates/move-compiler/src/naming/ast.rs

```diff
@@ -1210,11 +1210,11 @@ impl AstDebug for ModuleDefinition {
         w.writeln(match target_kind {
```

Comment:
Not necessary, but adding an AST Debug implementation for TargetKind this might make sense

Reply:
Done!